### ISSUE TITLE
SharedInformers: Update event continuously received

### ISSUE THREAD

**Issue by akram** [fabric8io/kubernetes-client#2977]
Hi team,

Using 5.1.2 ou 5.3.0,  after updating an object that I am watching, I see update events continuously appearing at a random pace of 30s or 60s even if resync is set to 0.
The method `onUpdate` of my informer implementation get called every 30s or 60s, while the following message appears only once:
```
io.fabric8.kubernetes.client.informers.cache.ReflectorWatcher - Event received MODIFIED
```
Additional logs here: 
https://pastebin.com/jhEzr0Vm

**Comment by hanikesn** [comment]
Are you positive that your Pods and status don't get updated periodically on the api server side?

**Comment by akram** [comment]
In my real situation scenario, I am seeing that with `ConfigMaps` that I do not update for sure.

**Comment by rohanKanojia** [comment]
@akram : Hi, I think this is a possible duplicate of #3016 . Could you please confirm if it's the case? If yes, shall we close this one?
